packages/tabs/src/containers/TabsContainer.example.md

```diff
@@ -1,82 +1,7 @@
-```jsx
-initialState = {
-  tabs: ['Tab 1', 'Tab 2', 'Tab 3'],
-  selectedKey: 'Tab 1',
-  focusedKey: undefined
-};
+## DEPRECATION WARNING
 
-<TabsContainer
-  selectedKey={state.selectedKey}
-  focusedKey={state.focusedKey}
-  onStateChange={newState => setState(newState)}
->
-  {({ getTabListProps, getTabProps, getTabPanelProps }) => (
-    <TabsView>
-      <TabList {...getTabListProps()}>
-        {state.tabs.map(tab => (
-          <Tab
-            {...getTabProps({
-              key: tab,
-              selected: tab === state.selectedKey,
-              focused: tab === state.focusedKey
-            })}
-          >
-            {tab}
-          </Tab>
-        ))}
-      </TabList>
-      {state.tabs.map(tab => (
-        <TabPanel
-          {...getTabPanelProps({
-            key: tab
-          })}
-        >
-          {tab} content
-        </TabPanel>
-      ))}
-    </TabsView>
-  )}
-</TabsContainer>;
-```
+This component has been deprecated in favor of the API provided in the
+[@zendeskgarden/container-tabs](https://www.npmjs.com/package/@zendeskgarden/container-tabs)
+package.
```

Comment:
Should you keep the examples here until it's removed?

Reply:
Sure, I will add that back.